packages/hw-app-ada/test/src/current/integration/getExtendedPublicKey.js

```diff
@@ -0,0 +1,78 @@
+import { expect } from "chai";
+import pathDerivations from "../../pathDerivations";
+
+import { getAda, pathToArray } from "../../utils";
+
+describe("getExtendedPublicKey", async () => {
+  let ada = {};
+
+  beforeEach(async () => {
+    ada = await getAda();
+  });
+
+  afterEach(async () => {
+    await ada.t.close();
+  });
+
+  it("Should successfully get extended public key", async () => {
```

Comment:
Does this work as expected?
In jest we had to declare upfront the number of assertions in the async case in orther for the checking to work but maybe Mocha is different ...

Reply:
Looks to be working properly